package.json

```diff
@@ -5,25 +5,38 @@
   "type": "module",
   "exports": {
     ".": {
-      "sass": "./scss/graupl.scss"
+      "sass": "./scss/graupl.scss",
+      "import": "./dist/js/graupl.es.js",
+      "require": "./dist/js/graupl.cjs.js"
     },
     "./*.scss": {
       "sass": "./scss/*.scss"
     },
     "./src/*.scss": {
       "sass": "./src/scss/*.scss"
+    },
+    "./carousel": {
+      "import": "./dist/js/component/carousel.es.js",
+      "require": "./dist/carousel.cjs.js"
+    },
+    "./alert": {
+      "import": "./dist/js/component/alert.es.js",
+      "require": "./dist/js/component/alert.cjs.js"
     }
   },
   "scripts": {
     "prepare": "husky",
     "commit": "git cz",
     "eslint": "eslint .",
-    "prettier": "prettier '**/*.{js,mjs,cjs,scss,sass}'",
-    "stylelint": "stylelint '**/*.scss' --allow-empty-input",
+    "prettier": "prettier 'src/**/*.{js,mjs,cjs,scss,sass}'",
+    "stylelint": "stylelint 'src/**/*.scss' --allow-empty-input",
```

Comment:
Everything needs to be linted other than the dist directory. Do not change this.

Reply:
Okay I changed this back. I am not sure wht to do about this stylelint message though.

```
src/scss/utilities/spacing/_variables.scss
  1:1  ✖  Unknown rule @stylistic/string-quotes                      @stylistic/string-quotes
```

There are 596 of them and they are all unkowns rule messages from the sass.